src/Riok.Mapperly.Abstractions/MapperIgnoreMemberAttribute.cs

```diff
@@ -0,0 +1,10 @@
+using System.Diagnostics;
+
+namespace Riok.Mapperly.Abstractions;
+
+/// <summary>
+/// Ignores a member from the mapping.
+/// </summary>
+[AttributeUsage(AttributeTargets.Property | AttributeTargets.Field, AllowMultiple = true)]
+[Conditional("MAPPERLY_ABSTRACTIONS_SCOPE_RUNTIME")]
+public sealed class MapperIgnoreMemberAttribute : Attribute;
```

Comment:
Naming to be discussed.
IMO `MapperIgnoreAttribute` could be a name which would better match expectations as other serialization frameworks (e.g. SystemTextJson, Newtonsoft Json) use `JsonIgnoreAttribute` too.

`MapperIgnoreAttribute` was previously used by Mapperly (deprecated with v3.0 and removed with v3.4). I think it should be safe to reuse the same attribute name as due to the `AttributeUsage` is different (only Method was allowed before).